README.md

```diff
@@ -213,7 +213,10 @@ exec 1>&2
 
 # Run Runic on added and modified files
 mapfile -t files < <(git diff-index --name-only --diff-filter=AM master | grep '\.jl$')
-julia --project=@runic -m Runic --check --diff "${files[@]}"
+
+if [ ! -z "${files}" ]; then
```

Comment:
```suggestion
if [ ${#files[@]} -gt 0 ]; then
```

Reply:
(https://www.shellcheck.net/ complained about `${files}` only expanding the first element (although that would be fine in this case) and about the use of `! -z` instead of `-n` FYI)

Reply:
oh, TIL something new then. :)